R/rmarkdown.R

```diff
@@ -0,0 +1,41 @@
+
+#' Add an RMarkdown Template
+#'
+#' @param template_dir Name of the directory the template will live in within
+#' \code{inst/rmarkdown/templates}.
+#' @param template_name The name as printed in the template menu.
+#' @param template_description A short description of the template.
+#' @param template_create_dir Sets the value of \code{create_dir}
+#' in the template.yml
+#' @inheritParams use_directory
+#'
+#' @export
+#' @examples
+#' \dontrun{
+#' use_rmarkdown_template()
+#' }
+use_rmarkdown_template <- function(template_dir = "newtemplate",
+                                   template_name = "Template Name",
+                                   template_description = "A description of the template",
+                                   template_create_dir = FALSE,
+                                   base_path = ".") {
+
+  # Process some of the inputs
+  template_create_dir <- if(isTRUE(template_create_dir)) "true" else "false"
```

Comment:
I think this could just be `as.character(template_create_dir)`

Reply:
I was originally concerned it needed to be lowercase but I see the yaml standard is fine with upper case `TRUE` and `FALSE` so have made this change.